spec/lib/chess/node_spec.rb

```diff
@@ -0,0 +1,25 @@
+require './spec/spec_helper'
+
+RSpec.describe Chess::Node do
+  describe '#initialize' do
```

Comment:
There is no need IMO to test a class constructor, because a constructor shouldn't have any complex logic. It just receives attributes and sets them
```suggestion
  describe '.new' do
```